### PR TITLE
PI Metrics + PI bug fix

### PR DESCRIPTION
**Pull request by codyborn**
Fix bug in `PriceFeed.sol` where it previously compared inverted rates.

Add new PI metrics to the dashboard and to the network stats.
image

### CODE REVIEW

File: packages/dev-frontend/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -18,21 +18,33 @@ export const Dashboard = (): JSX.Element => (
...
         </Container>
-        <Container variant="twoThirds">
-          <Chart />
+        <Container variant="oneThird">
+          <DeviationFactorCard />
         </Container>
+        {/* <Container variant="oneThird">
+          <BorrowingFee />
+        </Container> */}
+        {/* <Container variant="oneThird">
```
Comment (iwooden) [inline packages/dev-frontend/src/pages/Dashboard.tsx:38]: Are these still meant to be commented out?

**Review by iwooden — APPROVED** [review]
Looks great! Only thing is that from the screenshot it looks like `Redemption Price` and `Redemption Rate` under `Network Stats` don't match the figures in the cards - may be confusing for users. Otherwise, the info & tooltips look awesome!

File: packages/dev-frontend/src/pages/Dashboard.tsx (hunk 1 above)
Comment (codyborn) [reply packages/dev-frontend/src/pages/Dashboard.tsx:38]: Yeah, these are some less relevant stats that I removed as it looked a bit crowded.

**Comment by codyborn** [comment]
> Looks great! Only thing is that from the screenshot it looks like `Redemption Price` and `Redemption Rate` under `Network Stats` don't match the figures in the cards - may be confusing for users. Otherwise, the info & tooltips look awesome!

Fixed the `Redemption Price`. Forgot to take the inverse. The `Redemption Rate` is correct.. it's just different than the `LED APY` stat because it's per-second instead of per-year.

**Pull request merged**